backend/api/models.py

```
@@ -80,4 +81,18 @@
+class Video(models.Model):
+    title = models.CharField(max_length=200)
+    description = models.TextField(blank=True)
+    category = models.CharField(max_length=100, blank=True)
+    visibility = models.CharField(max_length=20, default='private')
+    video_url = models.URLField()
+    thumbnail_url = models.URLField(blank=True)
+    upload_date = models.DateTimeField(auto_now_add=True)
+    uploader = models.ForeignKey(User, on_delete=models.CASCADE, related_name='videos')
+    class Meta:
+        db_table = 'videos'
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)